docs/images/devtools-golang-v1beta1.md

```diff
@@ -34,17 +37,17 @@ configuration must be compatible.
 
 Interface variables are configurable with environment variables.
 
-If any of the following files are found they will be loaded in the order they are
-listed in before any variables are defined:
+If any of the following files are found they will be loaded in the order they
+are listed in before any variables are defined:
```

Comment:
Unrelated reformatting

Reply:
My auto formatter just fixed the line lengths to be the enforced 80 character limit